### ISSUE TITLE
Error when creating group submission without student group

### ISSUE THREAD

**Issue by amelsmajic** [Sapphire-CM/Sapphire#582]
When I try to create a group submission without having an assigned student group I land on the nginx error page because of a null reference to the missing student group.

Not sure if it should be left like that or if it shouldn't be possible at all to click on the group submission if you're missing the student group.

**Comment by matthee** [comment]
Nah, we should not show an error page. To me, this indicates a missing validation in the model layer. 

Could you take a swing @amelsmajic?
